stopcovid/dialog/models/state.py

```diff
@@ -83,7 +83,6 @@ class PromptStateSchema(Schema):
     slug = fields.Str(required=True)
     start_time = fields.DateTime(required=True)
     failures = fields.Int(allow_none=True)
-    reminder_triggered = fields.Boolean(allow_none=True)
```

Comment:
Are we sure that removing this from the schema isn't going to cause errors for dicts that are already in dynamo?

Reply:
I think so, but good call I'll double check that

Reply:
very good catch actually, marshmallow blows up when you pass it extra fields. I'll add this back

Reply:
👍